spec/system/admin/organizations_system_spec.rb

```diff
@@ -151,5 +153,15 @@
       expect(page).to have_content("Default email text")
       expect(page).to have_content("Users")
     end
+
+    it "can edit a user within an organization" do
```

Comment:
I don't think we need a full system spec for this - you can just validate the button in the HTML in a request spec.